architecture/additional_concepts/authorization.adoc

```diff
@@ -352,31 +364,25 @@ values.
 
 ==== RunAsUser
 
-. MustRunAs - requires a `runAsUser` to be configured. Uses the configured `runAsUser` as the default.
-Validates against the configured `runAsUser`.
-. MustRunAsRange - requires `min` and `max` to be defined if not pre-allocated. Uses `min` as the default.
-Validates against the entire allowable range.
-. MustRunAsNonRoot - requires that the pod be submitted with a non-zero `runAsUser` or have the
-`USER` directive defined in the image.  No default provided.
-. RunAsAny - no default provided.  Allows any `runAsUser` to be specified.
+. `MustRunAs` - requires a *runAsUser* to be configured. Uses the configured *runAsUser* as the default.  Validates against the configured *runAsUser*.
+. `MustRunAsRange` - requires min and max to be defined if not pre-allocated. Uses min as the default.  Validates against the entire allowable range.
+. `MustRunAsNonRoot` - requires that the pod be submitted with a non-zero *runAsUser* or have the _USER_ directive defined in the image.  No default provided.
+. `RunAsAny` - no default provided.  Allows any *runAsUser* to be specified.
 
 ==== SELinuxContext
 
-. MustRunAs - requires `seLinuxOptions` to be configured if not using pre-allocated values, uses
-`seLinuxOptions` as the default, validates against `seLinuxOptions`.
-. RunAsAny - no default provided.  Allows any `seLinuxOptions` to be specified.
+. `MustRunAs` - requires _seLinuxOptions_ to be configured if not using pre-allocated values, uses _seLinuxOptions_ as the default, validates against _seLinuxOptions_.
+. `RunAsAny` - no default provided.  Allows any _seLinuxOptions_ to be specified.
 
 ==== SupplementalGroups
 
-. MustRunAs - requires at least one range to be specified unless using pre-allocated values.
-Uses the min value of the first range as the default.  Validates against all ranges.
-. RunAsAny - no default provided.  Allows any `supplementalGroups` to be specified.
+. `MustRunAs` - requires at least one range to be specified unless using pre-allocated values.  Uses the min value of the first range as the default.  Validates against all ranges.
+. `RunAsAny` - no default provided.  Allows any _supplementalGroups_ to be specified.
 
 ==== FSGroup
 
-. MustRunAs - requires at least one range to be specified unless using pre-allocated values.
-Uses the min value of the first range as the default.  Validates against all ranges.
-. RunAsAny - no default provided.  Allows any `fsGroup` to be specified.
+. `MustRunAs` - requires at least one id value to be specified unless using a pre-allocated value. Validates against the id.
```

Comment:
this desc can be the same as the sup group MustRunAs strategy.  They are the same thing and can be configured the same way (to allow ranges). It just happens that, by default, we are only using the single id.